src/query/interpreter.cpp

```diff
@@ -167,6 +167,51 @@ void memgraph::query::CurrentDB::CleanupDBTransaction(bool abort) {
 }
 // namespace memgraph::metrics
 
+struct QueryLogWrapper {
+  std::string_view query;
+  const memgraph::storage::PropertyValue::map_t *metadata;
+  std::string_view db_name;
+};
+
+#if FMT_VERSION > 90000
```

Comment:
Why do we need this if in toolchain we use `FMT_VERSION=10.1.1` and I would assume we won't regress and old toolchain doesn't work anymore.